Core/ShaderAnalysis/src/shae.py

```diff
@@ -324,6 +326,10 @@ def GetBinEncodingText(self, comment):
         foundBinCode = re.search(r' *[0-9a-fA-F]+: *([0-9a-fA-F]{8})( *[0-9a-fA-F]{8})?', comment)
         if foundBinCode:
             result = comment[comment.find(':')+1 :].lstrip(' ')
+        # Attempt to match comments containing only the bin code.
```

Comment:
Did you intend to replace the logic here?  It almost seems like maybe there is a missing "else:" statement after line 328.

Reply:
Thanks, yes I should have had an 'else' in there